pkg/controller/jobs/rayjob/rayjob_controller.go

```diff
@@ -88,6 +91,13 @@ func (j *RayJob) GVK() schema.GroupVersionKind {
 	return gvk
 }
 
+func (j *RayJob) PodLabelSelector() string {
+	if j.Status.RayClusterName != "" {
+		return fmt.Sprintf("%s=%s", batchv1.JobNameLabel, j.Name)
```

Comment:
```suggestion
		return fmt.Sprintf("%s=%s", rayutils.RayClusterLabelKey, j.Status.RayClusterName)
```

No, it's still not correct. It should be `j.Status.RayClusterName` here because rayjobs running on RayCluster. Please double check.

Reply:
This can help you https://kueue.sigs.k8s.io/docs/tasks/run/rayjobs/#example-rayjob.

Reply:
Thanks for pointing it out, I missed it somehow :/
Done https://github.com/kubernetes-sigs/kueue/pull/2280/commits/dcd5bc8e68d5dd6010d3121308b4c75536fc822b